src/app/(account)/schemas.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+
+export const emailSchema = z
+    .string()
+    .min(1, { message: 'Please enter your email' })
+    .email({ message: 'Please enter a valid email' });
+
+export const passwordSchema = z
+    .string()
+    .min(1, { message: 'Please enter a password' })
+    .min(8, { message: 'Password must be at least 8 characters' })
+    .regex(/^(?=.*[a-zA-Z]).+$/, { message: 'Password must include a letter' })
+    .regex(/^(?=.*[0-9]).+$/, { message: 'Password must include a number' });
+
+export const firstNameSchema = z
+    .string()
+    .min(1, { message: 'Please enter your first name' })
+    .regex(/^[a-zA-Z]+$/, {
+        message: 'Please enter a valid first name',
+    });
+
+export const lastNameSchema = z
+    .string()
+    .min(1, { message: 'Please enter your last name' })
+    .regex(/^[a-zA-Z]+$/, {
+        message: 'Please enter a valid last name',
+    });
+
+export const codeSchema = z
+    .string()
+    .min(1, { message: 'Please enter the code' })
+    .min(6, { message: 'Code must be 6 digits' })
+    .regex(/^\d+$/, {
+        message: 'Code must be numeric',
```

Comment:
Yea that's right but I want to keep each rules seperate. For length that's the rule above. This regex is just to keep it to be a numeric string.

Reply:
That's fair. I'd then just change `.min(6, { message: 'Code must be 6 digits' })` to be `.length(6, { message: 'Code must be 6 digits' })` since I assume the code must be 6 digits exactly, not *at least* 6 digits.

Reply:
The code is exactly 6 digits, so it should use ".length".